lib/active_model/serializer.rb

```diff
@@ -366,9 +366,10 @@ def associations(include_directive = ActiveModelSerializers.default_include_dire
     def serializable_hash(adapter_options = nil, options = {}, adapter_instance = self.class.serialization_adapter_instance)
       adapter_options ||= {}
       options[:include_directive] ||= ActiveModel::Serializer.include_directive_from_options(adapter_options)
-      if (fieldset = adapter_options[:fieldset])
```

Comment:
minor refactoring: this minor trick didn't help improve code readability or brevity